debian/patches/0001-airbrake.patch

```diff
@@ -0,0 +1,111 @@
+Description: Remove airbrake
+Author: Jessica Frazelle <jess@docker.com>
+Last-Update: 2015-01-12
+Forwarded: not-needed
+Origin: vendor
+---
+ examples/hook/hook.go      | 35 ------------------------------
+ hooks/airbrake/airbrake.go | 54 ----------------------------------------------
+ 2 files changed, 89 deletions(-)
+ delete mode 100644 examples/hook/hook.go
+ delete mode 100644 hooks/airbrake/airbrake.go
+
+Index: golang-logrus/examples/hook/hook.go
+===================================================================
+--- golang-logrus.orig/examples/hook/hook.go
++++ /dev/null
+@@ -1,35 +0,0 @@
+-package main
+-
+-import (
+-  "github.com/Sirupsen/logrus"
```

Comment:
The spacing here still looks like it's spaces, not tabs like upstream has. :wink: (which is why it won't apply)

I think you need to spend a little more time with `quilt` or `git diff` to get this dialed in. :heart: